.github/workflows/ci.yml

```diff
@@ -43,6 +43,12 @@ jobs:
       max-parallel: 10
       matrix:
         php: ['7.2', '7.3', '7.4', '8.0']
+        psr7: ['^1.7']
+        include:
+          - php: '7.4'
+            psr7: '^2.0'
+          - php: '7.4'
+            psr7: 'dev-master'
```

Comment:
`^2.0@dev` does this already, and is robust again branch renames.

Reply:
The PR is updated